cargo/cargo_build_script_runner/bin.rs

```diff
@@ -0,0 +1,60 @@
+// Copyright 2018 The Bazel Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// A simple wrapper around a build_script execution to generate file to reuse
+// by rust_library/rust_binary.
+extern crate cargo_build_script_output_parser;
+
+use cargo_build_script_output_parser::BuildScriptOutput;
+use std::env;
+use std::fs::File;
+use std::io::Write;
+use std::process::{exit, Command};
+use std::path::PathBuf;
+
+fn absolute_path(path: &str) -> PathBuf {
+    let mut p = env::current_dir().unwrap();
+    p.push(path);
+    p
+}
+
+fn main() {
+    let mut args = env::args().skip(1);
+    // Because of the Bazel's sandbox, bazel cannot provide full path, convert all relative path to correct path.
```

Comment:
Rather than relying on this assumption, how about just using: https://doc.rust-lang.org/std/fs/fn.canonicalize.html ?

Reply:
Done.